src/app/pages/Home/OtherComponents/Tokens/Tokens.tsx

```diff
@@ -39,6 +44,23 @@ export const Tokens: FC = () => {
     return searchFocused && searchValueExist && filteredAssets[activeIndex] ? filteredAssets[activeIndex] : null;
   }, [filteredAssets, searchFocused, searchValueExist, activeIndex]);
 
+  const tokensView = useMemo<Array<JSX.Element>>(() => {
+    const tokensJsx = filteredAssets.map(assetSlug => (
+      <ListItem
+        key={assetSlug}
+        assetSlug={assetSlug}
+        active={activeAssetSlug ? assetSlug === activeAssetSlug : false}
+        balances={latestBalances}
+      />
+    ));
+
+    tokensJsx.splice(1, 0, <PartnersPromotion key="promo-token-item" variant={PartnersPromotionVariant.Text} />);
```

Comment:
Did you forget to add a conditional for the case ads are disabled?

Reply:
It is included into `PartnersPromotion` component